### DISCUSSION TITLE
Please include a way to access defi-cli

### DISCUSSION THREAD

**Discussion by John-Gee** [DeFiCh/app#1144] [Ideas]
Hey,

since the GUI wallet obviously uses defi-cli inside, it'd be nice to also be able to access it from outside the wallet to automate stuff. For now it seems we would need to download the node on top of the wallet, but it seems redundant...

Thanks!

**Comment by izzycsy** [comment]
Please refer to [DeFiCh/ain](https://github.com/DeFiCh/ain)

**Comment by John-Gee** [comment]
Yes I'm aware of that as I wrote in the original post, but it seems
redundant.
